nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/renditions/RenditionsObject.java

```diff
@@ -0,0 +1,157 @@
+import java.util.Set;
+import java.util.function.Supplier;
+
+import javax.ws.rs.FormParam;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.nuxeo.ecm.automation.OperationException;
+import org.nuxeo.ecm.core.api.LockHelper;
+import org.nuxeo.ecm.core.bulk.BulkService;
+import org.nuxeo.ecm.core.bulk.message.BulkCommand;
+import org.nuxeo.ecm.core.bulk.message.BulkStatus;
+import org.nuxeo.ecm.core.bulk.message.BulkStatus.State;
+import org.nuxeo.ecm.platform.picture.recompute.RecomputeViewsAction;
+import org.nuxeo.ecm.platform.thumbnail.action.RecomputeThumbnailsAction;
+import org.nuxeo.ecm.webengine.model.WebObject;
+import org.nuxeo.ecm.webengine.model.impl.DefaultObject;
+import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.runtime.kv.KeyValueService;
+import org.nuxeo.runtime.kv.KeyValueStore;
+
+/**
+ * @since 11.1
+ */
+@WebObject(type = "renditions")
+public class RenditionsObject extends DefaultObject {
+
+    public static final String THUMBNAILS_DEFAULT_QUERY = "SELECT * FROM Document WHERE ecm:mixinType = 'Thumbnail' AND thumb:thumbnail/data IS NULL AND ecm:isVersion = 0 AND ecm:isProxy = 0 AND ecm:isTrashed = 0";
+
+    public static final String PICTURES_DEFAULT_QUERY = "SELECT * FROM Document WHERE ecm:mixinType = 'Picture' AND picture:views/*/title IS NULL";
+
+    public static final String RECOMPUTE_VIEWS_ACTION_COMMAND_ID = RecomputeViewsAction.ACTION_NAME + "_commandId";
+
+    public static final String RECOMPUTE_THUMBNAILS_ACTION_COMMAND_ID = RecomputeThumbnailsAction.ACTION_NAME
+            + "_commandId";
+
+    protected Set<State> notAtWork = EnumSet.of(State.UNKNOWN, State.ABORTED, State.COMPLETED);
+
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("pictures/recompute")
+    public BulkStatus doPostPictures(@FormParam("query") String query) throws OperationException {
+        BulkService bulkService = Framework.getService(BulkService.class);
+        String commandId = getCommandId(RECOMPUTE_VIEWS_ACTION_COMMAND_ID);
+        query = query != null ? query : PICTURES_DEFAULT_QUERY;
+        String finalQuery = !query.isEmpty() ? query : PICTURES_DEFAULT_QUERY;
+
+        Supplier<BulkStatus> supplier = () -> {
+            if (needToRun(bulkService, commandId)) {
+                String newCommandId = bulkService.submit(new BulkCommand.Builder(RecomputeViewsAction.ACTION_NAME,
+                        finalQuery).user(ctx.getPrincipal().getName())
+                                   .param(RecomputeViewsAction.PARAM_XPATH, "file:content")
+                                   .build());
+                putCommandId(RECOMPUTE_VIEWS_ACTION_COMMAND_ID, newCommandId);
+                return bulkService.getStatus(newCommandId);
+            }
+            throw new WebApplicationException(Response.Status.CONFLICT);
```

Comment:
You could use a static import for `Response.Status.CONFLICT` (detail, but usually we prefer it as it's more readable).